common/views/themes/typography.ts

```diff
@@ -146,7 +146,9 @@ export const typography = css<GlobalStyleProps>`
     &:hover {
       text-decoration: none;
     }
+  }
 
+  * {
```

Comment:
Since this is just applying to `*`, should it be moved to `wellcome-normalize`?

Reply:
good point – it's not _really_ typography specific. Will update